Rethinking/Data/bangladesh.csv

```diff
@@ -0,0 +1,3869 @@
+"woman";"district";"use.contraception";"living.children";"age.centered";"urban"
+
+1;1;0;4;18.44;1
+
```

Comment:
please remove blank lines :-)

Reply:
This again?? I swear I don't know where they come from :laughing: Will do ASAP :wink: